### PR TITLE
users recieved

### PR DESCRIPTION
**Pull request by Hm255**
get-users test

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -69,6 +69,79 @@ afterAll(() => {
...
+    // it('should return a 400 if the return is of an invalid type', () => {
+    //   return request (app)
+    //   .get('/api/users/46578')
+    //   .expect(400)
+    //   .then(({body}) => {
+    //     console.log(body)
+    //     expect(body.msg).toBe('ID is not valid (type is wrong)')
+    //   })
+    //})
+    it('should return a 404 if the return doesnt exist', () => {
```
Comment (MRombi) [inline __tests__/app.test.js]: This test is not needed as putting a number after / would represent another path. We should remove it 🙂

File: __tests__/app.test.js [hunk 2]
```diff
@@ -69,6 +69,79 @@ afterAll(() => {
...
+          {
+            username: 'dav3rid',
+            name: 'dave',
+            avatar_url:
+              'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png'
+          }
+        ]);
+      });
+    })
+    // it('should return a 400 if the return is of an invalid type', () => {
```
Comment (MRombi) [inline __tests__/app.test.js]: Please, remove code that is commented out 🙂

File: __tests__/app.test.js [hunk 3]
```diff
@@ -69,6 +69,79 @@ afterAll(() => {
   });
 
 
+  describe("GET/api/users", () => {
```
Comment (MRombi) [inline __tests__/app.test.js:72]: This test is not necessary as the endpoint should return all the users, please remove the test 🙂

**Review by MRombi — COMMENTED** [review]
Well done on this endpoint, I think you complicated it for yourself, you just needed one test for this endpoint so please remove the ones I have highlighted, good to merge once you have removed them! 🦾

File: __tests__/app.test.js (hunk 3 above)
Comment (Hm255) [reply __tests__/app.test.js:72]: tests have been removed, I will merge my work now

**Pull request merged**